binding/python/README.md

```diff
@@ -45,7 +62,7 @@ self.W = theano.shared(
 )
 ```
 
-If you want to use multiverso, you can modify them like this.
+If you want to use multiverso, you can modify them like this. `mv_shared` is just a wrapper of theano.shared. It acts same as theano.shared, but it give you convenient interface to sync values.
```

Comment:
Maybe it's better to add backticks (`) around

`theano.shared`

just like this.